ToDoList-backend/ToDoList.Api/BucketTasks/Models/BucketUpsertTaskDto.cs

```diff
@@ -12,7 +10,8 @@ public class BucketUpsertTaskDto
     public int BucketTaskStateId { get; set; }
     public BucketTaskPriority? BucketTaskPriority { get; set; }
     public int BucketTaskPriorityId { get; set; }
+    public Bucket? Bucket { get; set; }
     public int BucketId { get; set; }
-    public List<AssigneeDto> Assignee { get; set; } = new List<AssigneeDto>();
-
+    public Assignee? Assignee { get; set; }
```

Comment:
W dto do dodawania/edycji nie powinniśmy mieć w ogóle `Bucket` ani `Assignee`. Powinniśmy mieć wyłącznie ich ID. To ich ID dostajemy w requescie do API. Ogólnie pousuwajmy z tych UpsertDto propertiesy, ktorych nie uzywamy, i zostawmy tylko te, ktore faktycznie sa nam potrzebne. To normalne, że DTO i model bazodanowy różnią sie od siebie. Dlatego w ogóle je oddzielamy od siebie ;)

Reply:
Jasne, w tak małym projekcie jeszcze tego nie widać choć może jak się rozrośnie to trzeba będzie bardziej dywersyfikować model bazodanowy od różnych dto.
Jeśli chodzi o sam problem to commit na podstawie poprzedniego komentarza powinien rozwiązać ten bubel.